host.json

```diff
@@ -5,20 +5,21 @@
     "version": "[4.0.0, 5.0.0)"
   },
   "extensions": {
-    "tracing": {
-      "traceInputsAndOutputs": false,
-      "traceReplayEvents": false
-    },
     "http": {
       "routePrefix": ""
+    },
+    "blobs": {
+      "maxDegreeOfParallelism": 1,
+      "poisonBlobThreshold": 1
     }
   },
-  "functions": [ "Info", "QueueFdrJsonToXmlEventProcessor", "JsonErrorRetry" ],
   "logging": {
     "logLevel": {
       "default": "Error",
-      "Function.QueueFdrJsonToXmlEventProcessor": "Information",
-      "Function.JsonErrorRetry": "Information"
+      "Function.Info": "Information",
```

Comment:
are you sure to set the info level?

Reply:
Yes, but only for `BlobEventProcessor`, in case everything works without exceptions are 2 logs per BlobTrigger. it is preferable to keep these info logs to track the activity of the function.

For application-insight sampling is enabled.